Index: ossim_plugins/pdal/ossimPdalBridge.cpp
===================================================================
--- ossim_plugins/pdal/ossimPdalBridge.cpp	(revision 23190)
+++ ossim_plugins/pdal/ossimPdalBridge.cpp	(working copy)
@@ -53,10 +53,10 @@
        const std::string driver = factory.inferReaderDriver(fname);
        if (driver == "")
            throw pdal::pdal_error("File type not supported by PDAL");
-       pdal::Reader* reader = m_manager->addReader(driver);
+       pdal::Stage& reader = m_manager->addReader(driver);
        pdal::Options opts;
        opts.add("filename", fname);
-       reader->setOptions(opts);
+       reader.setOptions(opts);
    }
 
    //pdal::Stage* stage = m_manager->addFilter("filters.stats", m_manager->getStage());
Index: ossim_plugins/pdal/ossimPdalBridge.h
===================================================================
--- ossim_plugins/pdal/ossimPdalBridge.h	(revision 23190)
+++ ossim_plugins/pdal/ossimPdalBridge.h	(working copy)
@@ -7,6 +7,7 @@
 #include <pdal/PipelineManager.hpp>
 #include <pdal/PipelineReader.hpp>
 #include <pdal/StatsFilter.hpp>
+#include <pdal/Reader.hpp>
 
 class ossimPdalBridge
 {
